### ISSUE TITLE
I can't register  new user to get started thraugth web console

### ISSUE THREAD

**Issue by mfbasha** [bhdresh/Dejavu#65]
<img width="1283" alt="Screen Shot 2024-02-15 at 1 41 13 PM" src="https://github.com/bhdresh/Dejavu/assets/16327093/24e99d08-1235-4dea-b182-b1ab7cca4da9">
<img width="1283" alt="Screen Shot 2024-02-15 at 1 42 05 PM" src="https://github.com/bhdresh/Dejavu/assets/16327093/b618103a-b3b3-4b1f-9fe3-8849fc760f91">
<img width="1283" alt="Screen Shot 2024-02-15 at 1 43 00 PM" src="https://github.com/bhdresh/Dejavu/assets/16327093/17c59c7d-2549-46a6-9383-1fb9f7b8f387">

I can't complete the registeration however i can login to the console ssh 
(1) when i load https://192.168.56.109/ --> redirects to https://192.168.56.109/Decoify/loginView.php  
(2) i tried to load https://192.168.56.109/Decoify/registerView.php mannually 
(3) I fill all fields and click "get started" to redirect to loginView.php again 
(4) loginView.php always give "invalid username/passwor"

**Comment by nguyenthanhvu240** [comment]
nah,i do not have Organization Name field,just login first time,and that is your account.i used dejavu 15

**Comment by drawabdo0** [comment]
hi @mfbasha, hi @nguyenthanhvu240,
Can you please provide the OVA and Vmk images of Dejavu console and engine if you still have them, because the Camolabs.io is down
